platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/SwirldsPlatform.java

```diff
@@ -1571,6 +1640,105 @@ public void startSyncNetwork() {
         }
     }
 
+    /**
+     * Starts sync as a protocol network, with work managed by a {@link NegotiatorThread}
+     * <p>
+     * Starting sync with this method supports emergency reconnect
+     *
+     * @param connectionManagers the constructed connection managers
+     */
+    private void startSyncAsProtocolNetwork(@NonNull final StaticConnectionManagers connectionManagers) {
+        final BasicConfig basicConfig = platformContext.getConfiguration().getConfigData(BasicConfig.class);
+        final SyncConfig syncConfig = platformContext.getConfiguration().getConfigData(SyncConfig.class);
+
+        final Duration hangingThreadDuration = basicConfig.hangingThreadDuration();
+
+        // if this is a single node network, start dedicated thread to "sync" and create events
+        if (initialAddressBook.getSize() == 1) {
+            syncProtocolThreads.add(new StoppableThreadConfiguration<>(threadManager)
+                    .setPriority(Thread.NORM_PRIORITY)
+                    .setNodeId(selfId.getId())
+                    .setComponent(PLATFORM_THREAD_POOL_NAME)
+                    .setOtherNodeId(selfId.getId())
+                    .setThreadName("SingleNodeNetworkSync")
+                    .setHangingThreadPeriod(hangingThreadDuration)
+                    .setWork(new SingleNodeNetworkSync(
+                            this::checkPlatformStatus,
+                            eventTaskCreator::createEvent,
+                            this::getSleepAfterSync,
+                            selfId.getId()))
+                    .build(true));
+
+            return;
+        }
+
+        // If we still need an emergency recovery state, we need it via emergency reconnect.
+        // Start the helper now so that it is ready to receive a connection to perform reconnect with when the
+        // protocol is initiated.
+        if (emergencyRecoveryManager.isEmergencyStateRequired()) {
+            reconnectController.get().start();
+        }
+
+        final PeerAgnosticSyncChecks peerAgnosticSyncChecks = new PeerAgnosticSyncChecks(List.of(
+                () -> !gossipHalted.get(), () -> intakeQueue.size() < settings.getEventIntakeQueueThrottleSize()));
+
+        for (final NodeId otherId : topology.getNeighbors()) {
```

Comment:
there seems to be overlap between this and the chatter protocol building. is there a reason why they don't share the same code but use different protocols?